.changeset/config.json

```diff
@@ -1,10 +1,10 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
   "changelog": ["@changesets/changelog-github", { "repo": "drwpow/openapi-typescript" }],
+  "ignore": ["openapi-typescript-docs"],
   "commit": false,
   "fixed": [],
   "linked": [],
   "access": "public",
-  "baseBranch": "main",
-  "updateInternalDependencies": "patch"
```

Comment:
Can be removed, as this currently blocks releases. `openapi-fetch` doesn’t rely on `openapi-typescript` except as a devDep for testing. Release-wise it’s safe to be out-of-date